libs/interfaces/ListFile.ts

```diff
@@ -76,50 +76,4 @@ export interface ListFileOptions {
  *
  * @see {@link https://docs.imagekit.io/api-reference/media-api/list-and-search-files#response-structure-and-status-code-application-json}
  */
-export interface ListFileResponse {
```

Comment:
I have made sure that the ListFileResponse is not being used anywhere else

Reply:
But FileObject is being used in many places and it would be incorrect now
![Uploading image.png…]()

Reply:
Yes, FileObject is being used at other places, but it's being used in getFileDetails, getFileVersions, getFileVersionDetails, updateFileDetails, restoreFileVersion. 

All the above functions deal with File and FileVersion